doc/eventhandler/index.rst

```diff
@@ -36,6 +36,38 @@ all further API calls.
 
    *An action is bound to the event* token_init.
 
+Pre and Post Handling
+---------------------
+
+.. index:: Pre Handling, Post Handling
+
+Added in Version 2.23.
+
+With most event handlers you can decide if you want the action to be taken before the actual event or
+after the actual event. I.e. if all conditions would trigger certain actions the action is either triggered
+before (*pre*) the HTTP request is processed or after (*post*) the request is processed.
```

Comment:
After a night of sleep :-) I think that the documentation would already be clearer if we replace "HTTP request" with "API request". Because pre-event handlers do not actually trigger before the HTTP request is handled (pre-policies are already handled at that point). What do you think?

Reply:
+1

Reply:
done